src/astHelpers.ts

```diff
@@ -218,3 +218,7 @@ export function matchesModifier(node: ts.Node, flags: ts.ModifierFlags) {
   const nodeFlags = ts.getCombinedModifierFlags(node as any);
   return (nodeFlags & flags) === flags;
 }
+
+export const isNamespaceDeclaration = (node: ts.ModuleDeclaration) => {
+  return node.modifiers && (node.name.pos - node.modifiers[node.modifiers?.length - 1].end) === 10 /* "namespace".length + 1 */
```

Comment:
I really don’t like this magic number counting, for example it is not resilient to whitespace and comments in between the modifiers, and `node.name`. I’m sure there is a better way to handle this somehow.